File: testprojects/scala_main/pipelines/DONOT_OPEN_SCALA/code/src/main/scala/org/main/scla_dep_mgmt/graph/SchemaTransform_1.scala
package org.main.scla_dep_mgmt.graph

import io.prophecy.libs._
import org.main.scla_dep_mgmt.udfs.UDFs._
import org.main.scla_dep_mgmt.config.Context
import org.apache.spark._
import org.apache.spark.sql._
import org.apache.spark.sql.functions._
import org.apache.spark.sql.types._
import org.apache.spark.sql.expressions._
import java.time._

object SchemaTransform_1 {

  def apply(context: Context, in: DataFrame): DataFrame =
    in.withColumn("c_concat_new_short_decimal",
                  concat(col("`c_struct-c_short`"), col("`c_struct-c_decimal`"))
      )
      .drop("c_array_long")
      .withColumnRenamed("c_array_boolean", "c_array_boolean_renamed")

}

File: testprojects/master_python_mainone/pipelines/StreamingPipelineMain/code/streamingpipelinemain/graph/SchemaTransform_1.py
from pyspark.sql import *
from pyspark.sql.functions import *
from pyspark.sql.types import *
from prophecy.libs import typed_lit
from streamingpipelinemain.config.ConfigStore import *
from streamingpipelinemain.udfs.UDFs import *

def SchemaTransform_1(spark: SparkSession, in0: DataFrame) -> DataFrame:
    return in0.withColumn("c_new_col", concat(col("c_boolean"), col("c_float")))
